src/scipp/signal/__init__.py

```diff
@@ -86,15 +77,29 @@ def butter(coord: Variable, *, N: int, Wn: Variable, **kwargs) -> SOS:
 
 
 @wrap1d(keep_coords=True)
-def sosfiltfilt(da: DataArray, dim: str, *, sos: SOS, **kwargs) -> DataArray:
+def _sosfiltfilt(da: DataArray, dim: str, *, sos: SOS, **kwargs) -> DataArray:
+    if not identical(da.coords[dim], sos.coord):
```

Comment:
Does `identical` compare if the coords have the same buffer? If not, doing the comparison here when `obj` is a Variable below would be unnecessarily expensive.

Reply:
It compares the buffer points in that cases, i.e., this should be fast.